contracts/V1/StateChannelDiamondProxy/AStateChannelManagerProxy.sol

```diff
@@ -3,34 +3,38 @@ pragma solidity ^0.8.8;
 import "./StateChannelCommon.sol";
 import "../StateChannelManagerInterface.sol";
 import "./StateChannelUtilLibrary.sol";
+import "./ConsumerFacet.sol";
 
 import "./DisputeManagerFacet.sol";
 import "./FraudProofFacet.sol";
 import "./DisputeFraudProofFacet.sol";
 import "./StateSnapshotFacet.sol";
 import "./JoinChannelFacet.sol";
 
-abstract contract AStateChannelManagerProxy is StateChannelManagerInterface, StateChannelCommon {
+contract AStateChannelManagerProxy is StateChannelManagerInterface, StateChannelCommon {
```

Comment:
`StateChannelManagerProxy`

Reply:
this is a big change that causes circular dependency issue
ill put it in separate pr to not muddy the water